react-example/src/views/InApp.tsx

```diff
@@ -95,7 +95,7 @@ export const InApp: FC<{}> = () => {
   return (
     <>
       <h1>inApp Endpoints</h1>
-      <Heading>POST /inApp/getMessages</Heading>
+      <Heading>POST /inApp/web/getMessages</Heading>
```

Comment:
This is also in the react-example app, minimal change to text that I think we can just keep.